controllers/openshift_controller.go

```diff
@@ -940,8 +963,16 @@ func (r *KataConfigOpenShiftReconciler) processKataConfigInstallRequest() (ctrl.
 		r.kataConfig.Status.InstallationStatus.IsInProgress = "false"
 		err := r.createRuntimeClass()
 		if err != nil {
-			return ctrl.Result{}, err
+			// Give sometime for the error to go away before reconciling again
+			return reconcile.Result{Requeue: true, RequeueAfter: 15 * time.Second}, err
+		}
```

Comment:
I don't understand this part. Is this needed for the rest to succeed or is this an improvement/fix ?

Reply:
This is improvement. The errors are transient and this introduces a short delay before trying the reconcile loop.